sentry-android-core/src/test/java/io/sentry/android/core/ActivityFramesTrackerTest.kt

```diff
@@ -341,6 +342,35 @@ class ActivityFramesTrackerTest {
         assertNull(sut.takeMetrics(fixture.sentryId))
     }
 
+    @Test
+    fun `start, setMetrics and stop call to FrameMetricsTracker is done on the main thread, even when being called from a background thread`() {
+        val sut = fixture.getSut()
+
+        // .addActivity
+        val callAddThread = Thread {
+            sut.addActivity(fixture.activity)
+        }
+        callAddThread.start()
+        callAddThread.join(500)
+        verify(fixture.handler, times(1)).post(any())
```

Comment:
You can omit `times(1)` because its the default.

Reply:
❤️ , you still remember after 2,5 years @marandaneto 😄